pkg/jsonschema/json-schema_test.go

```diff
@@ -283,7 +287,7 @@ func TestSampleInput(t *testing.T) {
 
 			input, err := os.ReadFile(tc.filePath)
 			require.NoError(t, err)
-			var m interface{}
+			var m any
```

Comment:
I'm curious - this change is just a style thing?

Reply:
yes